### PR TITLE
[NF] Add profiles for specific usage

### PR DESCRIPTION
**Pull request by arnaudbore**

### CODE REVIEW

**Review by mdesco — APPROVED** [review]
Lets test on tracto-inferno/harmonization and then merge if you agree

File: USAGE [hunk 1]
```diff
@@ -203,6 +203,11 @@ ABS                                         When this profile is used, TractoFlo
                                             This profile must be used for pathological data.
                                             The aparc+aseg.nii.gz and wmparc.nii.gz must be in the same space than t1.nii.gz
 
+bundling                                    When this profile is used, it will activate custom tracking parameters to improve recobundle results.
+                                            Local tracking will be enable with fa seeding mask.
```
Comment (GuillaumeTh) [inline USAGE]: with fa seeding and tracking mask

**Review by GuillaumeTh — APPROVED** [review]
One small comment

File: USAGE (hunk 1 above)
Comment (arnaudbore) [reply USAGE]: Tracking mask is already the case no?
I just want to emphasize on the parameters we change with this profil.

Comment (GuillaumeTh) [reply USAGE]: No. Tracking mask and seeding mask are separated and are both set as WM by default

**Pull request merged**